cfme/tests/infrastructure/test_relationships.py

```diff
@@ -0,0 +1,80 @@
+from utils import testgen
+import pytest
+
+"""
+This test can run only after overcloud cloud provider created and linked to
+undercloud infra provider, need to compare the cloud providers with the
+results of the relationships
+"""
+
+pytestmark = [pytest.mark.meta(server_roles='+smartproxy +smartstate')]
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = \
+        testgen.infra_providers(metafunc, required_fields=["ssh_credentials"])
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist,
+                        scope="module")
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_assinged_roles(provider, soft_assert):
+    provider.load_details()
+    result = int(provider.summary.relationships.deployment_roles.value)
```

Comment:
`.value` already converts the text value to a number it possible, so this is unnecessary.